src/_guides/language/language-tour.md

```diff
@@ -3853,17 +3829,16 @@ Click the run button ( {% img 'red-run.png' %} ) to execute the code.
 https://gist.github.com/405379bacf30335f3aed
 https://dartpad.dartlang.org/405379bacf30335f3aed
 
-<?code-excerpt "language-tour/callable-function/bin/main.dart"?>
+<?code-excerpt "lib/language_tour/callable_classes.dart"?>
 {% prettify dart %}
 class WannabeFunction {
-  void call(String a, String b, String c) => a + ' ' + b + ' ' + c + '!';
+  String call(String a, String b, String c) => '$a $b $c!';
 }
 
-void main() {
-  var wf = new WannabeFunction();
-  var out = wf("Hi", "there,", "gang");
-  print('$out');
-}
+var wf = new WannabeFunction();
+var out = wf("Hi", "there,", "gang");
```

Comment:
" -> '

While you're in there... The sample code should use `'` instead of `"`, for consistency (except where we're discussing `"` specifically).

Reply:
Done for this example. And I double checked that I caught that needed switch in all other examples (except where it was more convenient to use "... like 'this' ...").